package.json

```diff
@@ -69,6 +69,7 @@
     "conventional-commit-types": "^2.1.0",
     "copy-webpack-plugin": "^1.1.1",
     "cz-conventional-changelog": "^1.2.0",
+    "d3": "^3.5.17",
```

Comment:
I think it does, but I was copying patterns from an earlier project to get this working here.

Would you prefer if I upgrade to D3 v4 right in this PR, or defer to a future PR?